filebeat/input/tcp/conn.go

```diff
@@ -0,0 +1,72 @@
+package tcp
```

Comment:
I've keep theses methods public, because I might move it to common.

Reply:
We should only move it to common if we also use it in `metricbeat` for example, which could happen.